core/node/app_registry/sync/tracked_stream.go

```diff
@@ -64,6 +65,7 @@ func (b *AppRegistryTrackedStreamView) onNewEvent(ctx context.Context, view *Str
 		return err
 	}
 	apps := mapset.NewSet[string]()
```

Comment:
Streams of type `STREAM_USER_INBOX_BIN` are added to the tracked set of streams in this PR. There is a special check on line 54/55 if the tracked stream is of this type. If so calls `processUserInboxMessage` and returns. Any logic after this check isn't executed.

Is it not enough to just start tracking these user inbox streams?

Reply:
Thanks the comment. I decided to remove the code that adds `STREAM_USER_INBOX_BIN` to the set of tracked streams, now I'm only adding app user inbox streams to be tracked. Previously, all user inbox streams would be tracked if I got it right